tasty-plutus/src/Test/Tasty/Plutus/Internal/Context.hs

```diff
@@ -106,13 +112,32 @@ data ExternalType
       Show
     )
 
+{- | Different types of value:
+
+  `TokensValue` is only used within `ContextBuilder ('ForMinting r)`
+  for representing classes, belonging to the tested MintingPolicy.
+
+  In all other cases `GeneralValue` is used.
+
+ @since 6.0
+-}
+data ValueType
+  = -- | @since 6.0
+    GeneralValue Value
+  | -- | @since 6.0
+    TokensValue TokenName Positive
+  deriving stock
+    ( -- | @since 6.0
+      Show
+    )
+
 {- | An input to a script, consisting of a value and a type.
 
  @since 1.0
 -}
 data Input
   = -- | @since 3.0
-    Input ExternalType Value
+    Input ExternalType ValueType
```

Comment:
Since this constructor is exported, this is an API change. Therefore, the `since` needs to match the version where it changed.